cmd/registry-replacer/main.go

```diff
@@ -75,6 +75,9 @@ func replacer(
 		}
 
 		for idx := range config.Images {
+			if config.Images[idx].From != "" {
```

Comment:
We should determine what field in the `Build` is set with the `images[*].from` and what that behavior is -- it might replace only the "first" `FROM` in the Dockerfile, in which case we stlll need to mess with the others.

Reply:
Done